crate/utils/src/crypto/cover_crypt/encryption.rs

```diff
@@ -135,7 +131,7 @@ impl EncryptionSystem for CoverCryptEncryption {
         );
         Ok(EncryptResponse {
             unique_identifier: self.public_key_uid.clone(),
-            data: Some(ciphertext),
+            data: Some(ciphertext.to_vec()),
```

Comment:
We actually may want to have two serializable traits: `Serializable` and `SecureSerializable`. That way encrypted objects would be serialized directly to `Vec<u8>` rather than `Zeroizing<Vec<u8>>`, which would avoid needing this clone.

Reply:
creating crypto_core issue: https://github.com/Cosmian/crypto_core/issues/51